### ISSUE TITLE
training on custom data plus taking a pretrained class

### ISSUE THREAD

**Issue by ektaraj** [experiencor/keras-yolo2#425]
I want to train the model on say ten custom classes but also want person to be detected too. How can I retain the pretrained class like person and include new classes?

Can just adding person to the labels help without adding any person example in the custom dataset?
